contracts/SpokePool.sol

```diff
@@ -1519,7 +1498,7 @@ abstract contract SpokePool is
         uint256 returnValue;
         bytes memory data = abi.encodeCall(IERC20Upgradeable.transfer, (to, amount));
         assembly {
-            success := call(gas(), token, 0, add(data, 0x20), mload(data), 0, 0x20)
+            success := call(MAX_ERC20_TRANSFER_GAS_COST, token, 0, add(data, 0x20), mload(data), 0, 0x20)
```

Comment:
Since each leaf is a separate token, shouldn't we just assume that the executor will protect themselves against this and not execute leaves where the token is expensive to transfer? You could even imagine the executor having a token whitelist where it refuses to pay out tokens that are not on that list.

Reply:
thats a good point too, and reduces contract complexity +1

Reply:
https://github.com/across-protocol/contracts/pull/926/commits/263dcd915b9a6d2e9aba7b88f6691d7be020e1d6